pytorch_lightning/plugins/training_type/ipu.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         device_iterations: int = 1,
         autoreport: bool = True,
         autoreport_dir: Optional[str] = None,
-        parallel_devices: Optional[List[torch.device]] = None,
+        parallel_devices: Optional[List[torch.device]] = (),
```

Comment:
This is a bit hacky, I changed this to an empty tuple as this will prevent a crash since parallel devices haven't been defined when the plugin is passed and no `ipus` have also been defined in the Trainer.